index.html

```diff
@@ -26,7 +26,8 @@ <h1>Binge Pad</h1>
     <!-- static class attribute 
     <div class="tv-static"></div>
     -->
-
+    <script src="test-helpers.js"></script>
```

Comment:
This is great, thank You for adding the script and src for the test helpers where there ready 🐰